versions.json

```diff
@@ -1,38 +1,34 @@
 [
   {
-    "version": "6.9.1",
-    "static_tag": "stable"
+    "version": "6.6.41",
```

Comment:
Similar to https://github.com/cilium/ci-kernels/pull/66#discussion_r1757374693, could we keep the order of the versions in the file? Or is this an issue with the regeneration script? Might need some jq magic added to sort the entries.

Reply:
> Or is this an issue with the regeneration script?

This is what the script does. I can try to update it so it sorts by kernel version

Reply:
That would be great, thank you! :pray:

Reply:
Turns out the script was sorting as a string, changed it now to do proper semantic version sorting